webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  entry: { content: "./src/content.ts", background: "./src/background.ts" },
```

Comment:
src statt Src

Reply:
nice that it works anyway xD ... will use src for the folder